### DISCUSSION TITLE
如何在参考文献中加入编号 很多老师要求这个

### DISCUSSION THREAD

**Discussion by zijuzhang** [whutug/whu-thesis#198] [General]
预答辩时老师要求论文的参考文献页之前需要添加编号
下图示例是不行的
![image](https://user-images.githubusercontent.com/14990346/228174016-208f6143-f51d-4788-b589-e3ba13650d87.png)

**Comment by sikouhjw** [comment]
拿规范去怼老师或者用 biblatex 并设置著录样式 `bibstyle` 为顺序编码值。

**Reply by zijuzhang** [reply]
没法啊 三个不同学院的人 都被老师这么说  好奇怪

**Reply by note286** [reply]
https://gs.whu.edu.cn/info/1022/3231.htm 这里附页五（1）给了示例了啊，就是没有序号的。

**Reply by zijuzhang** [reply]
啊 这。。。那就不改了？好吧 还说我们目录二级标题三级标题无缩进

**Reply by note286** [reply]
你去研究生院下载最新的规定，对着看一下，以这个为准，如果老师有疑问，你就把规定给他看。

**Reply by sikouhjw** [reply]
> 没法啊 三个不同学院的人 都被老师这么说 好奇怪
>啊 这。。。那就不改了？好吧 还说我们目录二级标题三级标题无缩进

我说的很明确了，拿规范怼，或者用 biblatex。

并且模板提供了接口 `style/bib-style={author-year-numbered}`，请去读文档。
https://github.com/whutug/whu-thesis/blob/6f19c2e0d34bbfc752f86a7ea505154a185ff595/whu-thesis-doc.tex#L473-L478

**Comment by note286** [comment]
可以参考 https://github.com/nju-lug/NJUThesis/issues/151 和 https://github.com/nju-lug/NJUThesis/discussions/182 。

**Reply by zijuzhang** [reply]
完了。。。我发现我不知道在哪里搞。。。

**Reply by note286** [reply]
你需要简单了解一下`biblatex`。
